tests/models/llama/test_tokenization_llama.py

```diff
@@ -546,6 +546,15 @@ def test_special_token_special_word(self):
         decoded_tokens = tokenizer.decode(input_ids)
         self.assertEqual(decoded_tokens, " <s> Hello<s> how")
 
+    def test_some_edge_cases(self):
```

Comment:
I suggest another edge case with the string ">". I'm still afraid it will tokenize as `["<", "unk", ">>"]` and then cut off the leading three elements, resulting in `[]` (?).

Reply:
The string `'>'` does not start with `SPIECE_UNDERLINE` so will just be tokenized with the sp model to `>`. 
Wil add it too 😉